rpcs3/Emu/RSX/GL/GLGSRender.h

```diff
@@ -9,6 +9,7 @@
 #include "GLOverlays.h"
 #include "../rsx_utils.h"
 #include "../rsx_cache.h"
+#include "../Common/texture_cache.h"
```

Comment:
GLTextureCache.h is included above which definately includes texture_cache.h. The explicit imports of rsx_utils and rsx_cache above this line are not needed either but likely predate the shared texture cache.